test/integration-legacy/runner-spec.test.js

```diff
@@ -250,6 +251,7 @@ describe('Runner', function () {
                             expect(cursor.ref).to.eql(runStore.ref);
 
                             expect(response.code).to.be(200);
+                            expect(reason.lookup(response.code).name).to.be(response.status);
```

Comment:
This should be the other way round. Actual things should come in the `expect` call, and expected results compared with them.